src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -596,6 +597,12 @@ class CodeBlockPart extends Disposable implements IChatResultCodeBlockPart {
 			}
 		}));
 
+		if (this._accessibilityService.isScreenReaderOptimized()) {
+			const toolbarElt = this.toolbar.getElement();
+			toolbarElt.style.display = 'block';
+			toolbarElt.ariaLabel = this._configurationService.getValue(AccessibilityVerbositySettingId.Chat) ? localize('chat.codeBlock.toolbarVerbose', 'Toolbar for code block which can be reached via tab') : localize('chat.codeBlock.toolbar', 'Code block toolbar');
```

Comment:
it's a little weird that in the tab order, the toolbar comes first. thus this aria label. improvements are to come though that will make this unnecessary 

https://github.com/microsoft/vscode-internalbacklog/issues/4168